src/main/java/org/jfrog/bamboo/task/ArtifactoryDeploymentUploadTask.java

```diff
@@ -29,10 +31,12 @@
  *
  * @author Aviad Shikloshi
  */
-public class ArtifactoryDeploymentUploadTask extends AbstractSpecTask implements DeploymentTaskType {
+public class ArtifactoryDeploymentUploadTask implements DeploymentTaskType {
```

Comment:
The *ArtifactoryDeploymentUploadTask* and *ArtifactoryDeploymentDownloadTask* should also report their usage.